apps/swirl-docs/src/components/Navigation/SidebarNavigation/SidebarNavItem.tsx

```diff
@@ -0,0 +1,64 @@
+import { NavItem } from "@swirl/lib/navigation/";
+import classNames from "classnames";
+import Link from "next/link";
+import { HttpMethods } from "oas/dist/rmoas.types";
+import { forwardRef, useRef } from "react";
+import {
+  Tag,
+  mapHttpMethodToTagContent,
+  mapHttpMethodToTagScheme,
+} from "../../Tags";
+
+export const SidebarNavItem = forwardRef<
```

Comment:
As those Nav Items are now used for both the mobile nav and the sidebar nav, the sidebar nav naming here is misleading. we should call them navigation items